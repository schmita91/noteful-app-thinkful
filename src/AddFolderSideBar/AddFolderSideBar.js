import React, {Component} from 'react'


class AddFolderSideBar extends Component {
    handleBack = () =>{
        this.props.history.goBack();
    }
    render() {
        return (
            <button className="back"
                    onClick={this.handleBack}
                    >Back</button>
        )
    }

}
export default AddFolderSideBar;