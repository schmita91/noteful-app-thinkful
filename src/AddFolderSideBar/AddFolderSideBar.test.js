import React from 'react';
import ReactDOM from 'react';
import AddFolderSideBar from './AddFolderSideBar';
import renderer from 'react-test-renderer';


describe('Add Folder Side Bar', ()=>{
    it('renders to UI as expected', ()=>{
        const tree = renderer.create(<AddFolderSideBar/>).toJSON
        expect(tree).toMatchSnapshot();
    })
})