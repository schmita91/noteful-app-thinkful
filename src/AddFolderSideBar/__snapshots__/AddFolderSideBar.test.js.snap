// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Folder Side Bar renders to UI as expected 1`] = `[Function]`;
